src/api/methods/getUnsignedTransferTx.js

```diff
@@ -1,6 +1,13 @@
+const { isValidAddress } = require('ethereumjs-util');
+const getColor = require('./getColor');
 const makeTransfer = require('../../txHelpers/makeTransfer');
 
-module.exports = async (bridgeState, from, to, color, value) => {
+module.exports = async (bridgeState, from, to, colorOrAddress, value) => {
+  let color = colorOrAddress;
+  if (isValidAddress(colorOrAddress)) {
+    color = await getColor(bridgeState, colorOrAddress);
```

Comment:
What if a token is not registered?

Reply:
Nice point. In this case, `getColor` will throw "Unknown token address" which is a way to go, I guess. I've added couple of tests for this just now